trade-finance-manager-api/src/v1/controllers/auth/auth-provider.js

```diff
@@ -0,0 +1,239 @@
+const msal = require('@azure/msal-node');
+const axios = require('axios');
+const { msalConfig, AZURE_SSO_TENANT_SUBDOMAIN, REDIRECT_URI, AZURE_SSO_POST_LOGOUT_URI } = require('./auth-provider-config');
+
+class AuthProvider {
+  constructor(config) {
+    this.config = config;
+    this.cryptoProvider = new msal.CryptoProvider();
```

Comment:
So there is a refactor we could do here at a semi-glance were we could update all these classes to me only static methods. Not sure if it's worth it

Reply:
I'm going to leave this for now